src/Softeq.NetKit.Chat.Tests/Integration/BaseTest.cs

```diff
@@ -63,8 +63,6 @@ private async Task CleanUpDatabase(ILifetimeScope lifetimeScope)
                 await connection.OpenAsync();
 
                 var sqlQuery = @"   
-                    DELETE FROM DirectChannelSettings;
```

Comment:
We need to apply this changes?

Reply:
yes, this implementation already removed